source/landing-zone.txt

```diff
@@ -9,9 +9,252 @@ Landing Zone Design
+
+Why Do You Need a Landing Zone?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Establishing a landing zone gives you confidence that
+every workload your team deploys aligns with your requirements around
+security, performance, reliability, operational efficiency, and cost
+optimization. You don't want developers on every team following different standards, and designing and sharing out a landing zone helps
+ensure alignment across all teams and stakeholders on how your
+organization will work in {+service+}.
+
+.. _landing-zone-considerations:
+
+What are some Important Landing Zone Considerations?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Consider the following organization-specific
+requirements when you create a landing zone:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 40
+
+   * - Consideration
+     - Description
+   
+   * - System Partition Requirements
+     - Identify how you will partition systems for management. For
+       example, clarify how your team should arrange {+service+}
+       projects or {+service+} organizations. 
+       
+       To get recommendations and learn more about this topic, see
+       :ref:`arch-center-hierarchy`.
+   * - Access Controls
+     - Identify MongoDB Atlas Control Plane access controls, and
+       database access controls for both workload and workforce principals. Create a comprehensive list of principals and mechanisms for how you will authenticate and authorize. Define Atlas API key access controls, including authorizations and expiration rules.
+   * - Change Control and Auditability Requirements
+     - Clarify any change control or audit controls requirements. This
+       can include change approval processes and tools, along with
+       reporting guidelines.
+
+       To get recommendations and learn more about this topic, see
+       :ref:`arch-center-auditing-logging`.
+   * - Data Residency Requirements
+     - Identify data residency requirements, such as data sovereignty
+       requirements, or application data partitioning rules.
+   * - Network Topology Requirements
+     - Identify network topology requirements, including Cloud Provider
+       regions, private connectivity options, and application
+       deployment topology.
+
+       To get recommendations and learn more about this topic, see
+       :ref:`arch-center-network-security`.
+   * - High Availability Requirements
+     - Identify high availability requirements such as cluster topology
+       with node count and priority per region, global write
+       requirements, and partitioning tolerances.
+
+       To get recommendations and learn more about this topic, see
+       :ref:`arch-center-high-availability`.
+   * - Data Retention Requirements
+     - Identify and record your data retention policies. This may
+       require creating a classification for automation, including
+       creation of archive and/or purge automation. In some cases, data
```

Comment:
Our [style guide](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/) discourages this construction as ambiguous. 
```suggestion
       creation of archive or purge automation. In some cases, data
```